dbos/_context.py

```diff
@@ -31,6 +31,9 @@ class OperationType(Enum):
 
 OperationTypes = Literal["handler", "workflow", "transaction", "step", "procedure"]
 
+MaxPriority = 2**31 - 1  # 2,147,483,647
```

Comment:
do we need such a large range?

Reply:
Maybe not -- this is just the maximum number of a signed integer.